src/algebra/ordered_monoid.lean

```diff
@@ -87,22 +124,55 @@ lemma mul_le_mul' (h₁ : a ≤ b) (h₂ : c ≤ d) : a * c ≤ b * d :=
 lemma mul_le_mul_three {e f : α} (h₁ : a ≤ d) (h₂ : b ≤ e) (h₃ : c ≤ f) : a * b * c ≤ d * e * f :=
 mul_le_mul' (mul_le_mul' h₁ h₂) h₃
 
+-- here we start using properties of one.
 @[to_additive le_add_of_nonneg_right]
 lemma le_mul_of_one_le_right' (h : 1 ≤ b) : a ≤ a * b :=
-have a * 1 ≤ a * b, from mul_le_mul_left' h _,
-by rwa mul_one at this
+by simpa only [mul_one] using mul_le_mul_left' h a
 
 @[to_additive le_add_of_nonneg_left]
 lemma le_mul_of_one_le_left' (h : 1 ≤ b) : a ≤ b * a :=
-have 1 * a ≤ b * a, from mul_le_mul_right' h a,
-by rwa one_mul at this
+by simpa only [one_mul] using mul_le_mul_right' h a
 
-@[to_additive lt_of_add_lt_add_right]
-lemma lt_of_mul_lt_mul_right' (h : a * b < c * b) : a < c :=
-lt_of_mul_lt_mul_left'
-  (show b * a < b * c, begin rw [mul_comm b a, mul_comm b c], assumption end)
+@[to_additive add_le_of_nonpos_right]
+lemma mul_le_of_le_one_right' (h : b ≤ 1) : a * b ≤ a :=
+by simpa only [mul_one] using mul_le_mul_left' h a
+
+@[to_additive add_le_of_nonpos_left]
+lemma mul_le_of_le_one_left' (h : b ≤ 1) : b * a ≤ a :=
+by simpa only [one_mul] using mul_le_mul_right' h a
+
+@[to_additive]
+lemma lt_of_mul_lt_left (h : a * b < c) (hle : 1 ≤ b) : a < c :=
+(le_mul_of_one_le_right' hle).trans_lt h
+
+@[to_additive]
+lemma lt_of_mul_lt_right (h : a * b < c) (hle : 1 ≤ a) : b < c :=
+(le_mul_of_one_le_left' hle).trans_lt h
+
+@[to_additive]
+lemma le_of_mul_le_left (h : a * b ≤ c) (hle : 1 ≤ b) : a ≤ c :=
```

Comment:
This lemma is very similar to `le_mul_of_one_le_of_le` below - so I think it would make sense to include `one_le` in its name.

Reply:
Should I add `one_le`/`le_one` to the names of the other lemmas then as well?